fileio/private/ft_hastoolbox.m

```diff
@@ -408,9 +410,7 @@
     dependency = {'bf.ttest', 'bf.ttest2'};
   case 'EZC3D'
     dependency = {'ezc3dRead', 'ezc3dWrite'};
-  case 'XSENS'
```

Comment:
why did you remove the XSENS support?

Reply:
No, we didn't try to do that.  Please see the comments above.